contracts/ocr2/src/integration_tests.rs

```diff
@@ -105,7 +110,6 @@ fn transmit_report(env: &mut Env, epoch: u32, round: u8) {
     };
     env.router
         .execute_contract(transmitter.clone(), env.ocr2_addr.clone(), &msg, &[])
```

Comment:
Probably easiest to get rid of `anyhow` and return `.is_ok()` here

Reply:
I think then I won't be able to assert the error is an invalid sig one right?